src/views/assets/stylesheets/Image.css

```diff
@@ -0,0 +1,17 @@
+.Image {
+  float: left;
+  width: 320px;
+  height: 350px;
+  margin: 20px;
+  padding: 20px;
+  background-color: #222;
+  color: white;
+}
+
+.Image > img {
+  max-width: 100%;
+  max-height: calc(100% - 1rem);
+}
+.Image.selected {
```

Comment:
Disclaimer: I know nothing about CSS 😓

- I'm a little bit confused about this one. Why the upper case class name? (I imagine it has smtg to do with the component name?)

- Is selected a name or a selector? I suppose that it's a class because you're using . instead of :, so why no space between both classes?